mcs/class/corlib/System.Threading/Thread.cs

```diff
@@ -89,6 +89,8 @@ sealed class InternalThread : CriticalFinalizerObject {
 		private IntPtr owned_mutex;
 		private IntPtr suspended_event;
 		private int self_suspended;
+		private int num_waiters;
+		private int freeing;
```

Comment:
I'd kind of like to see byte used for booleans instead of int32.

Reply:
Just use `bool` in C# and `mono_bool` in the matching struct in native.  (C# bool is 1 byte)